source/nodejs/adaptivecards-designer/src/toolbar.ts

```diff
@@ -95,7 +95,8 @@ export class ToolbarButton extends ToolbarElement {
     }
 
     protected internalRender(): HTMLElement {
-        let element = document.createElement("button");
+		let element = document.createElement("button");
+		element.type = "button"
```

Comment:
Can you elaborate on why this is necessary? As far as I know, `type` defaults to "button" for button tags.